Zotero/Scenes/AppCoordinator.swift

```diff
@@ -207,8 +211,12 @@ extension AppCoordinator: AppDelegateCoordinatorDelegate {
                     collection = Collection(custom: .all)
                 }
                 mainController.showItems(for: collection, in: data.libraryId)
-                guard data.restoreMostRecentlyOpenedItem, let item = data.openItems.first else { return }
-                restoreMostRecentlyOpenedItem(using: self, item: item)
+                guard data.restoreMostRecentlyOpenedItem else { return }
+                openItemsController.restoreMostRecentlyOpenedItem(using: self, sessionIdentifier: sessionIdentifier) { item in
+                    if item == nil {
+                        DDLogInfo("AppCoordinator: no open item to restore")
```

Comment:
You can add log for non-nil `item` too.

Reply:
Check https://github.com/zotero/zotero-ios/pull/829/commits/055044308a8c2ccae4bb61b6472cc1e9676a3c9d